addons/initialise-sprite-position/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Change new sprite default position",
-  "description": "Moves sprites to a specified x/y position when they are created.",
+  "name": "Customizable new sprite position",
+  "description": "Change the default x/y position new sprites are moved to when they are created.",
```

Comment:
Change the default x/y positions of new sprites.

Reply:
I'm not sure if "when they are created" should be kept or not, but I'll definitely remove "are moved to".